src/hotspot/share/c1/c1_Compiler.cpp

```diff
@@ -237,7 +237,7 @@ bool Compiler::is_intrinsic_supported(vmIntrinsics::ID id) {
   case vmIntrinsics::_counterTime:
 #endif
   case vmIntrinsics::_getObjectSize:
-#if defined(X86) || defined(AARCH64) || defined(S390) || defined(RISCV) || defined(PPC64)
+#if defined(X86) || defined(AARCH64) || defined(S390) || defined(RISCV64) || defined(PPC64)
```

Comment:
PS: The change of macro `RISCV` seems unrelated to this PR? Seem better to go with another PR.